uqcsbot/uptime.py

```diff
@@ -10,7 +11,9 @@
 
 
 class UpTime(commands.Cog):
-    CHANNEL_ID = 836243768411160606
+    # Checks for an Azure specific environment variable, if it exists we're running as prod.
+    CHANNEL_ID = 836243768411160606 if os.environ.get("WEBSITE_SITE_NAME") != None \
```

Comment:
Similar to my comments on /membercount, we should avoid using IDs wherever possible. See other cogs (e.g. starboard) for name-based handling of channels - it needs an on_ready handler, but in return, it works for anyone who might spin up the bot in their own servers, not just uqcs & testing.

Reply:
Completely agree. Have modified to correct for this.